README.md

```diff
@@ -60,27 +60,54 @@ pip install --user mal
 
 ### Manual Installation
 
-Clone this project and run inside it:
+Clone this project and inside it run:
 
 ```
 make install
 ```
 
-Probably we'll need have super-user permissions, but I'd recommend you
-to install inside of a virtualenv or use the `pip install --user' stuff.
+`mal` requires super-user permissions when you run `make install` inside of a `virtualenv`.
+We strong encourage you to install it with `pip install --user`.
 
 ### On ArchLinux
 
-If you're using the archlinux distro this project has been packaged and uploaded to
-the AUR as [python-mal-git](https://aur.archlinux.org/packages/python-mal-git).
+This project has been packaged and uploaded to the AUR as
+[python-mal-git](https://aur.archlinux.org/packages/python-mal-git) in case you're using an archlinux distro.
+
+You may install it using an AUR wrappers such `yaourt` or `pacaur`, making
```

Comment:
Please, remove that section about Arch Linux in details, I already remove that in earlier version. As I said for you, README.md was the doc more updated, I forgot to remove that from sphinx docs (please, remove from that too)